### PR TITLE
Menu and wipe wallet screen redesign RW-106

### PR DESCRIPTION
**Pull request by majsavli**

### CODE REVIEW

File: brd-android/app/src/main/res/layout/controller_wipe_wallet.xml [hunk 1]
```diff
@@ -1,77 +1,74 @@
...
+	<com.breadwallet.legacy.presenter.customviews.BaseTextView
+		android:id="@+id/textView5"
+		android:layout_width="0dp"
+		android:layout_height="wrap_content"
+		android:layout_marginHorizontal="60dp"
+		android:layout_marginTop="16dp"
+		android:fontFamily="@font/mobile_font_book"
+		android:text="@string/RecoveryKeyFlow.wipeWalletSubtext"
+		android:textAlignment="center"
+		android:textColor="@color/fabriik_shuttle_grey"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_wipe_wallet.xml:56]: Old color used here

File: brd-android/app/src/main/res/layout/controller_wipe_wallet.xml [hunk 2]
```diff
@@ -1,77 +1,74 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/white">
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:background="@color/white">
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_wipe_wallet.xml:6]: Check background color for this screen. It should be named `light_background_{something}`

File: brd-android/app/src/main/java/com/breadwallet/ui/settings/SettingsScreenHandler.kt [hunk 3]
```diff
@@ -268,37 +268,44 @@ class SettingsScreenHandler(
...
-                R.drawable.ic_feedback
+                R.drawable.ic_review,
+                R.color.light_text_03
             ),
             SettingsItem(
                 context.getString(R.string.MenuButton_support),
                 SettingsOption.SUPPORT,
-                R.drawable.ic_support
+                R.drawable.ic_support,
+                R.color.light_text_03
```
Comment (roknovak) [inline brd-android/app/src/main/java/com/breadwallet/ui/settings/SettingsScreenHandler.kt:284]: Is required to set the same color for each item? You could probably just change default color to `light_text_03`

Comment (majsavli) [reply brd-android/app/src/main/java/com/breadwallet/ui/settings/SettingsScreenHandler.kt:284]: I tried changing it in the settings_list_item layout, but nothing changed

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
         android:layout_marginTop="16dp"
         android:layout_marginBottom="16dp"
         android:fontFamily="@font/roboto_regular"
-        android:textColor="@color/fabriik_black"
+        android:textColor="@color/red"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:21]: Red? 🤔

File: brd-android/app/src/main/java/com/breadwallet/ui/settings/SettingsScreenHandler.kt (hunk 3 above)
Comment (roknovak) [reply brd-android/app/src/main/java/com/breadwallet/ui/settings/SettingsScreenHandler.kt:284]: SettingsItem constructor has default title color set to `light_text_01`. You should change this color to `light_text_03`

File: brd-android/app/src/main/res/layout/settings_list_item.xml (hunk 4 above)
Comment (majsavli) [reply brd-android/app/src/main/res/layout/settings_list_item.xml:21]: Uff, forgot to change it back, when i was testing it.

Comment (roknovak) [reply brd-android/app/src/main/res/layout/settings_list_item.xml:21]: Also, please add TextStyle to views in this layout

File: brd-android/app/src/main/res/layout/controller_wipe_wallet.xml [hunk 5]
```diff
@@ -1,77 +1,74 @@
...
-        android:layout_marginEnd="32dp"
-        android:fontFamily="@font/mobile_font_book"
-        android:text="@string/RecoveryKeyFlow.wipeWallet"
-        android:textAlignment="center"
-        android:textSize="24sp"
-        android:textColor="@color/fabriik_black"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/imageView5" />
+	<com.breadwallet.legacy.presenter.customviews.BaseTextView
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_wipe_wallet.xml:33]:
```suggestion
	<TextView
```

File: brd-android/app/src/main/res/layout/controller_wipe_wallet.xml [hunk 6]
```diff
@@ -1,77 +1,74 @@
...
-        android:layout_marginEnd="40dp"
-        android:fontFamily="@font/mobile_font_book"
-        android:text="@string/RecoveryKeyFlow.wipeWalletSubtext"
-        android:textAlignment="center"
-        android:textSize="16sp"
-        android:textColor="@color/fabriik_shuttle_grey"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView3" />
+	<com.breadwallet.legacy.presenter.customviews.BaseTextView
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_wipe_wallet.xml:47]:
```suggestion
	<TextView
```

File: brd-android/app/src/main/res/layout/controller_wipe_wallet.xml [hunk 7]
```diff
@@ -1,77 +1,74 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/white">
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:background="@color/light_outline_01">
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_wipe_wallet.xml:6]: Is this correct color? Outline colors are mostly used for separators

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 8]
```diff
@@ -1,71 +1,71 @@
...
-        tools:text="Display Currency" />
+	<TextView
+		android:id="@+id/item_title"
+		style="@style/TextStyle.Subtitle1"
+		android:layout_width="wrap_content"
+		android:layout_height="wrap_content"
+		android:layout_marginStart="16dp"
+		android:layout_marginTop="16dp"
+		android:layout_marginBottom="16dp"
+		android:textColor="@color/fabriik_black"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:21]: Incorrect color

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 9]
```diff
@@ -1,71 +1,71 @@
...
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="@+id/item_title"
-        tools:text="USD" />
+	<TextView
+		android:id="@+id/item_addon"
+		android:layout_width="wrap_content"
+		android:layout_height="wrap_content"
+		android:layout_marginEnd="28dp"
+		android:fontFamily="@font/roboto_regular"
+		android:textColor="@color/fabriik_shuttle_grey"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:35]: Incorrect color, use light_text_03

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 10]
```diff
@@ -1,71 +1,71 @@
...
-        tools:text="Fiat currency used to display transactions amounts."
-        tools:visibility="visible" />
+	<TextView
+		android:id="@+id/item_sub_header"
+		android:layout_width="0dp"
+		android:layout_height="wrap_content"
+		android:layout_marginEnd="28dp"
+		android:layout_marginBottom="8dp"
+		android:fontFamily="@font/mobile_font_book"
+		android:textColor="@color/fabriik_shuttle_grey"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:62]: Incorrect color, use light_text_03

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 11]
```diff
@@ -1,71 +1,71 @@
...
+		android:textSize="12sp"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:63]: Remove textSize property

File: brd-android/app/src/main/res/layout/settings_list_item.xml [hunk 12]
```diff
@@ -1,71 +1,71 @@
...
-        app:layout_constraintTop_toTopOf="@+id/item_title"
-        tools:text="USD" />
+	<TextView
+		android:id="@+id/item_addon"
+		android:layout_width="wrap_content"
+		android:layout_height="wrap_content"
+		android:layout_marginEnd="28dp"
+		android:fontFamily="@font/roboto_regular"
+		android:textColor="@color/fabriik_shuttle_grey"
+		android:textSize="@dimen/sub_header"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/settings_list_item.xml:36]: Remove textSize property

File: brd-android/app/src/main/res/layout/settings_list_item.xml (hunk 11 above)
Comment (majsavli) [reply brd-android/app/src/main/res/layout/settings_list_item.xml:63]: If I use body1 and remove textSize, it becomes the same size as the title ?

**Review by roknovak — APPROVED** [review]

**Pull request merged**